browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -615,7 +615,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 		for (var i in data.children) {
 			var child = data.children[i];
-			if (child.id === 'help')
+			if (child.id === 'help' || child.id === 'zoterorefresh')
```

Comment:
I don't like this. We should put additional property in the JSON like 'leftaligned: true' and check here - so it will be more generic

Reply:
Makes sense, I have add that json property and have proceed accordingly. The 'help' case is still needed, I have checked but, and since it comes directly from core, we do not have any other place where we detect if we are in presence of a container with a help button